src/metabase/pulse.clj

```diff
@@ -136,8 +138,8 @@
         (when (mi/can-read? instance)
           (link-card->text (assoc link-card :entity instance)))))))
```

Comment:
It's not clear `link-card->text` returns a part, so perhaps rename it to `link-card->text-part` or refactor `link-card->text` so it only returns the text like so:
```suggestion
          {:type :text
           :text (link-card->text (assoc link-card :entity instance))
```

Reply:
I chose to make `link-card->text-part` returning a part.

Reply:
ba6cefff13